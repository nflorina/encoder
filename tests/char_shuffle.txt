T
[
H
]
:
B
~
>
"
b
F
f
)
z
O
.
e
'
!
:
b
10
END
