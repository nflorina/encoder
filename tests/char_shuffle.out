0 21 0
TT[TTH[T]HT:]TB:T~BT>~T">Tb"TFbTfFT)fTz)TOzT.OTe.T'eT!'T:!Tb[
zT.OTe.T'eT!fTz)TO'T:!Tb[]TB:T~>Tb"TFTT[TTHBT>~T"[T]HT:bTfFT)
